sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Microsoft.Azure.WebJobs.Extensions.ServiceBus.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Microsoft.Azure.WebJobs" />
     <PackageReference Include="Microsoft.Extensions.Azure" />
     <PackageReference Include="Azure.Messaging.ServiceBus" />
+    <PackageReference Include ="Azure.Core" />
```

Comment:
We should be able to remove this once Azure.Messaging.ServiceBus releases with Azure.Core 1.34+

Reply:
Can we add a comment/issue to track it?

Reply:
Tracking: https://github.com/Azure/azure-sdk-for-net/issues/37629